### PR TITLE
Dev/adjust

### PR DESCRIPTION
**Pull request by jameskeith**

### CODE REVIEW

File: .idea/.gitignore [hunk 1]
```diff
@@ -0,0 +1,3 @@
+# Default ignored files
```
Comment (tamayok) [inline .idea/.gitignore]: Add `.idea/` to gitignore.

File: adjust/build.gradle [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+group = 'com.tealium.remotecommands'
+version = "1.0.0"
+
+android {
+    compileSdkVersion 30
+    buildToolsVersion "30.0.3"
+
+    defaultConfig {
+        minSdkVersion 21
+        targetSdkVersion 30
```
Comment (tamayok) [inline adjust/build.gradle:16]: Not sure if it matters, but are we bumping up `targetSdkVersion` to 30? I believe all the others are at 29 still.

File: adjust/src/main/java/com/tealium/remotecommands/adjust/AdjustRemoteCommand.kt [hunk 3]
```diff
@@ -0,0 +1,287 @@
...
+
+    public override fun onInvoke(response: Response?) {
+        response?.apply {
+            val commandList = splitCommands(requestPayload)
+            parseCommands(commandList, requestPayload)
+            send()
+        }
+    }
+
+    internal fun splitCommands(payload: JSONObject): Array<String> {
```
Comment (tamayok) [inline adjust/src/main/java/com/tealium/remotecommands/adjust/AdjustRemoteCommand.kt]: Why `internal` and not `private`?

File: adjust/src/main/java/com/tealium/remotecommands/adjust/AdjustRemoteCommand.kt [hunk 4]
```diff
@@ -0,0 +1,287 @@
...
+        val commandString = payload.optString(Commands.COMMAND_NAME, "")
+        return commandString.split(AdjustConstants.SEPARATOR).map {
+            it.trim().toLowerCase(Locale.ROOT)
+        }.toTypedArray()
+    }
+
+    internal fun parseCommands(commands: Array<String>, payload: JSONObject) {
```
Comment (tamayok) [inline adjust/src/main/java/com/tealium/remotecommands/adjust/AdjustRemoteCommand.kt]: Same as above.

File: app/src/main/assets/adjust.json [hunk 5]
```diff
@@ -0,0 +1,54 @@
+{
+  "config": {
+    "api_token": "wv6wqpoxsqv4",
```
Comment (tamayok) [inline app/src/main/assets/adjust.json]: Is this a test token? Might be best to change to `"<YOUR_TOKEN_HERE>"` ... or something similar?

**Review by tamayok — COMMENTED** [review]
Good work

File: .idea/.gitignore (hunk 1 above)
Comment (jameskeith) [reply .idea/.gitignore]: done. thanks

File: adjust/src/main/java/com/tealium/remotecommands/adjust/AdjustRemoteCommand.kt (hunk 3 above)
Comment (jameskeith) [reply adjust/src/main/java/com/tealium/remotecommands/adjust/AdjustRemoteCommand.kt]: good catch - was a c/p fail
updated to private.

File: app/src/main/assets/adjust.json (hunk 5 above)
Comment (jameskeith) [reply app/src/main/assets/adjust.json]: it is our token for the adjust/android app - agree it should have been removed.
done.

File: adjust/build.gradle (hunk 2 above)
Comment (jameskeith) [reply adjust/build.gradle:16]: as you say; don't believe it should make any difference, but it's typically recommended to target the latest available - so we should probably be bumping the rest up to v30 at some point anyway

**Pull request merged**